app/models/bulk_upload.rb

```diff
@@ -2,9 +2,14 @@ class BulkUpload < ApplicationRecord
   enum log_type: { lettings: "lettings", sales: "sales" }
 
   belongs_to :user
+  has_many :bulk_upload_errors
 
   after_initialize :generate_identifier, unless: :identifier
 
+  def year_combo
+    "#{year}/#{year - 2000 + 1}"
+  end
+
```

Comment:
This is in quite a few places now. Anyway we could extract it? Would having it in the form handler make sense?

Reply:
Where would you extract to? You suggested `FormHandler` however `BulkUpload` has zero knowledge of `FormHandler` or forms in general, nor should they have knowledge of what a form even is?